src/auth/auth.ts

```diff
@@ -507,7 +507,21 @@ window.forceSignIn = withErrorReporting('Error forcing sign in', async (token) =
 // extending Window interface to access Appcues
 declare global {
   interface Window {
-    Appcues: any;
+    Appcues?: {
```

Comment:
Nice!

Is there any "and there might be other fields" symbol we could put at the end (in case they add more to this later), or is that implied in typescript?

Reply:
We could do 
```typescript
interface Window {
  Appcues?: {
    identify: (userId: string, properties?: any) => void;
    // other specified properties...
    [key: string]: any;
  }
}
```
but I think that would sort of defeat the purpose of having a type definition - if you're worried about new properties breaking things a la API client, Typescript won't complain if the Appcues script ends up having additional fields/utility methods, we would just need to add them here with proper types to be able to use them elsewhere (which would be beneficial for usage anyways).